packages/x-components/src/components/location-injector.vue

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+  import { Prop, Provide, Component } from 'vue-property-decorator';
+  import { NoElement } from '../components/no-element';
+  import { FeatureLocation } from '../types';
+  import BaseEventButton from './base-event-button.vue';
+
+  /**
+   * Location Injector component.
+   * This component injects the location with value passed as prop.
+   *
+   * @public
+   */
+  @Component({
+    components: { BaseEventButton }
+  })
+  export default class LocationInjector extends NoElement {
```

Comment:
Can you rename it to `LocationProvider`? It is providing a location, not injecting it 😅